sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/WrapperSdkExceptionManager.java

```diff
@@ -0,0 +1,139 @@
+    }
+
+    /**
+     * Store the in-memory wrapper exception data to disk. This should only be used by a wrapper SDK.
+     *
+     * @param data  The data to be saved
+     * @param errorId The associated error UUID
+     */
+    public static void saveWrapperExceptionData(byte[] data, @NonNull UUID errorId) {
+        if (errorId == null) {
+            MobileCenterLog.error(Crashes.LOG_TAG, "Failed to save wrapper exception data: null errorId");
+            return;
+        }
+
+        sWrapperExceptionDataContainer.put(errorId.toString(), data);
+        File dataFile = getFile(errorId);
+        try {
+            StorageHelper.InternalStorage.writeObject(dataFile, data);
+        }
+        catch (IOException e) {
+            MobileCenterLog.error(Crashes.LOG_TAG, "Failed to save wrapper exception data to file", e);
+        }
+    }
+
+    /**
+     * Delete wrapper exception data from disk and store it in memory
+     *
+     * @param errorId    The associated error UUID
+     */
+    public static void deleteWrapperExceptionData(@NonNull UUID errorId) {
+        if (errorId == null) {
+            MobileCenterLog.error(Crashes.LOG_TAG, "Failed to delete wrapper exception data: null errorId");
+            return;
+        }
+
+        File dataFile = getFile(errorId);
+        byte[] loadResult = loadWrapperExceptionData(errorId);
+        if (loadResult == null) {
+            MobileCenterLog.error(Crashes.LOG_TAG, "Failed to delete wrapper exception data: data not found");
+            return;
+        }
+        StorageHelper.InternalStorage.delete(dataFile);
+    }
+
+    /**
+     * Load wrapper exception data into memory
+     *
+     * @param errorId   The associated error UUID
+     * @return The data loaded into memory
+     */
+    public static byte[] loadWrapperExceptionData(@NonNull UUID errorId) {
+        if (errorId == null) {
+            MobileCenterLog.error(Crashes.LOG_TAG, "Failed to load wrapper exception data: null errorId");
+            return null;
+        }
+
+        byte[] dataBytes = sWrapperExceptionDataContainer.get(errorId.toString());
+        if (dataBytes != null) {
+            return dataBytes;
+        }
+
+        File dataFile = getFile(errorId);
+
+        try {
+            dataBytes = StorageHelper.InternalStorage.readObject(dataFile);
+            if (dataBytes != null) {
+                sWrapperExceptionDataContainer.put(errorId.toString(), dataBytes);
+            }
+            return dataBytes;
+        }
+        catch (ClassNotFoundException | IOException e) {
+            MobileCenterLog.error(Crashes.LOG_TAG, "Cannot access wrapper exception data file " + dataFile.getName(), e);
+        }
+        return null;
+    }
+
+    /**
+     * Get a file object for wrapper exception data
+     *
+     * @param reportId   The associated error UUID
+     * @return The corresponding file object
+     */
+    private static File getFile(@NonNull UUID reportId) {
```

Comment:
Hmm.. Where is my comment?? I just mentioned the parameter name but it is gone. Weird thing happened on GitHub.
`errorId` instead of `reportId`?